flow/connectors/postgres/cdc.go

```diff
@@ -463,6 +478,13 @@ func PullCdcRecords[Items model.Items](
 			continue
 		}
 
+		if fetchedBytesCounter != nil {
+			fetchedBytesCounter.Add(ctx, int64(len(msg.Data)), metric.WithAttributeSet(attribute.NewSet(
+				attribute.String(otel_metrics.FlowNameKey, req.FlowJobName),
+				attribute.String(otel_metrics.DeploymentUidKey, peerdbenv.PeerDBDeploymentUID()),
```

Comment:
@serprex I was thinking maybe we should move the deploymentUID key when we create the resource in otel_manager.go.

Reply:
will create followup PR